pkg/interface/src/logic/subscription/global.ts

```diff
@@ -61,7 +61,7 @@ export default class GlobalSubscription extends BaseSubscription<StoreState> {
       this.subscribe('/updates', 'hark-chat-hook', true);
       this.dequeue();
 
-    }, 1500)
```

Comment:
I'm not sure I understand why there's a delay here in the first place. Don't you always want the context's connection limit to be the case, rather than waiting on this timer in the "high connection limit" and "fast response" cases?